cmd/kogito/command/converter/config_converter.go

```diff
@@ -30,6 +29,8 @@ const (
 	configMapSuffix     = "custom-properties"
 	createdByAnnonKey   = "createdBy"
 	createdByAnnonValue = "Kogito CLI"
+	// ConfigMapApplicationPropertyKey ...
```

Comment:
not sure if ...Key fits the purpose of this var, maybe ConfigMapAppPropsFIleName?

Reply:
done